core/templates/dev/head/editor/ExplorationSettings.js

```diff
@@ -45,6 +45,8 @@ oppia.controller('ExplorationSettings', [
       });
     }
 
+    $scope.activeInputName = '';
```

Comment:
Actually, given where this is used, I suggest getting rid of activeInputName altogether and just having a boolean variable: isRolesFormOpen.